src/components/title-block/TitleBlock.tsx

```diff
@@ -12,15 +13,23 @@ type TitleBlockProps = {
   img: string
   translationKey: string
   children?: React.ReactNode
+  sx: SxProps
+  mt: SxProps
```

Comment:
what is mt and sx, if u need pass styles via props to different component pass as object with keys for every component u need